qutebrowser/browser/webengine/webenginesettings.py

```diff
@@ -356,6 +376,17 @@ def init_private_profile():
         assert private_profile.isOffTheRecord()
         private_profile.setter.init_profile()
 
+        _qute_scheme_handler.install(private_profile)
+        _req_interceptor.install(private_profile)
+        _download_manager.install(private_profile)
+        cookies.install_filter(private_profile)
+
+        # Clear visited links on web history clear
+        history.web_history.history_cleared.connect(
+            default_profile.clearAllVisitedLinks)
```

Comment:
This should be `private_profile` :wink: Noticed too late, but I'm now working on getting rid of the duplication anyways.

Reply:
Ugh, copy-paste already wreaking havoc.